packages/office-addin-unit-test/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "office-addin-unit-test",
```

Comment:
I would suggest office-addin-mock or something like that.  'unit-test' is ambiguous about weather this is a test suit or the tool that we intend it to be.